scss/_skeletons.scss

```diff
@@ -0,0 +1,48 @@
+@import "variables";
+
+img:empty,
```

Comment:
has this been tested with existing images? i'm working on stream tiles right now and wondering how it will look since we have a shimmer class that shows an animation beforehand...

Reply:
Yeah, we had a lot of discussion around this and decided let's roll it out to all images. We can pull it back if it has some unintended consequence. Maybe we could end up adding the ability to "opt out" rather than "opt in."

Reply:
well i just pasted those styles in stream tiles to test and looks fine 🚀